src/main/app/utils/redirectHelper.ts

```diff
@@ -0,0 +1,38 @@
+import * as config from 'config'
+import * as express from 'express'
+import * as toBoolean from 'to-boolean'
+import { OAuthHelper } from 'idam/oAuthHelper'
+import { buildURL } from 'utils/callbackBuilder'
+import { Paths as AppPaths } from 'app/paths'
+import { Paths as FirstContactPaths } from 'first-contact/paths'
+import { RoutablePath } from 'common/router/routablePath'
+
+const oauthEnabled = toBoolean(config.get('featureToggles.idamOauth'))
+
+export class RedirectHelper {
```

Comment:
What do you think about providing two different classes instead of doing `if` logic in the methods?

Reply:
open to it but didn't think it was needed as we will be removing the original code in a sprint or so. can do if you think its worth it

Reply:
I thought the old logic will be kept around for longer. If it's removed next sprint then it's fine.

Reply:
am extracting anyway for the fun of it

Reply:
but yes should be removed quite soon